share/src/main/resources/alfresco/site-webscripts/org/alfresco/errorcode.lib.ftl

```diff
@@ -0,0 +1,15 @@
+<#-- MNT-20195 (LM-190214): strip out and return error code from given error message. -->
+<#function getErrorCode message>
+   <#assign code = "">
+
+   <#if message?? && message?is_string>
+      <!-- substring first 8 characters from message that usually be the error code id. -->
```

Comment:
This statement will always be true? Aren't there any scenarios where this statement might be wrong? I'm concerned that we might introduce a bug if there will be a change in the message.

Reply:
Looks like the build error log number will always be an 8 digit number (MMDDXX, where M is month, D is day and X is an atomic integer count) that will prefix the error message.

Reply:
Ok. Thank you for clarifications!